ddl/placement_policy.go

```diff
@@ -232,6 +230,33 @@ func onAlterPlacementPolicy(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 		return ver, errors.Trace(err)
 	}
 
+	ids, err := getPlacementPolicyDependedObjectsIDs(t, oldPolicy)
+	if err != nil {
+		return ver, errors.Trace(err)
+	}
+	if len(ids) != 0 {
+		// build bundle from new placement policy.
+		bundle, err := placement.NewBundleFromOptions(newPolicyInfo.PlacementSettings)
+		if err != nil {
+			return ver, errors.Trace(err)
+		}
+		err = bundle.Tidy()
+		if err != nil {
+			return ver, errors.Trace(err)
+		}
+		// Do the http request only when the rules is existed.
+		bundles := make([]*placement.Bundle, 0, len(ids))
+		for _, id := range ids {
```

Comment:
The id list contains database ids. However db do not have records stored in tikv

Reply:
Nice insight, the work is also a preparation for what is coming from @zhaoxugang / @Mattias